src/main/java/com/deavensoft/paketomat/email/EmailService.java

```diff
@@ -0,0 +1,17 @@
+package com.deavensoft.paketomat.email;
+
+
+
+import java.util.Map;
+
+public interface EmailService {
+    // Method
+    // To send a simple email
```

Comment:
If the name of the method is descriptive enough, comments are excessive. 

- [ ] Delete all redundant comments.